ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -196,6 +204,7 @@ export const DEVNET = `
         "targetBlockTimeInSeconds": 60,
         "targetBucketTimeInSeconds": 10,
         "maxBlockSizeBytes": 524288,
-        "minFee": 0
+        "minFee": 0,
+        "disallowNegativeBlockMineTime": 3
```

Comment:
Is there a specific reason for 3? I feel like 0 is the most logical choice here, otherwise we're always going to be wondering why 3 was chosen in the future (same with the strategy tests below)

If you need to modify it for the tests, you should be able to do something like `chain.consensus.parameters.negativeBlockMineTime = 3` in the beforeAll or something

Reply:
I use 3 because the sequence starts at 1. But 1 is genesis block so it has special logic and skip the time check. If we set to 1, this logic 
```
 this.chain.consensus.isActive(
        this.chain.consensus.parameters.disallowNegativeBlockMineTime,
        current.sequence,
      )
``` 
would never be called in devnet.
If I set to 2, it will be called but it will never go to the else statement since all the blocks <= the activation sequence. 
So I set to 3, so the first block is the old logic and the rest blocks are the new logics. I think this config is the most generic for devnet and unit tests. What is your thought on this?

Reply:
What is the goal of having it activate on devnet for 1 block? If the goal is ensure the code path is hit, I think that is the responsibility of unit tests. It's also ensured that its hit for both testnet and mainnet since both are guaranteed to have blocks with a negative time difference since the blockchains are immutable and we will forever have to support those blocks prior to this change.

Reply:
Updated the dev value to 1, and set the value for relevant unit tests